tier4_control_msgs/msg/WillMove.msg

```diff
@@ -0,0 +1,2 @@
+builtin_interfaces/Time stamp
+bool data
```

Comment:
When will this data be true/false?

Reply:
Whether the target velocity is zero or not. This is done for the command selected by the operation mode.

https://github.com/autowarefoundation/autoware.universe/pull/1810/files#diff-c6cb4d89b8200c23b9b1122c4fe09f7d632739fa755790bb015985d2ec4249b1R53-R56

https://github.com/autowarefoundation/autoware.universe/pull/1810/files#diff-e9575d30dc30d07fd3f0a0786f4264f368cd0747acef3e5fc77d1e50b1252a48R413

Reply:
I see. How about asking @maxime-clem about the naming?
From the word `WillMove`, I personally feel it's one time only.

Reply:
I am sorry I am not familiar with the `PauseInterface`.
What I understand:
- The command gate can be "paused" or "unpaused".
- If "unpaused", the control command is used.
- If "paused", the control command is no longer used.
- If "paused", a stopping command is used.
- In both cases, a message is published to indicate if the control command has a velocity of 0 or not.
What do you think about changing `WillMove` to something like `MovingControlCmd` or `StoppedControlCmd` (true if velocity = 0) ?

Reply:
@kenji-miyake @maxime-clem 
Thank you. That's right. This is a function that pauses the vehicle to insert processing when the vehicle starts. `IsPaused` is a pause state and `WillMove` is used to check if the vehicle is about to depart.

I feel `MovingControlCmd` is a command rather than a bool. So how do you feel about `IsRequestingDeparture` or `IsStartRequesting`?

Reply:
`IsRequestingDeparture` sounds good to me, but then should we also rename the state name? :thinking: 
Or `IsStartRequesting` might be acceptable.

Reply:
Sorry my suggestion was not good and I understand how it could be confused with a control command :disappointed: 
My issue with `IsRequestingDeparture` (and `IsStartRequesting`) is that if sounds like the command gate is the one requesting departure. However, the message informs about the state (requesting to move or not) of the _input_ of the command gate.

I would thus suggest to change to `isDepartureRequested`.